provider/testdata/sidecar_testdata.toml

```diff
@@ -1,7 +1,13 @@
 [frontends]
     [frontends.web]
-    backend = "web"
-    passHostHeader = true
-    entrypoints = ["http","https"]
+        backend = "web"
+        passHostHeader = true
+        entrypoints = ["http","https"]
         [frontends.web.routes.test_1]
-        rule = "Host: some-aws-host"
+            rule = "Host: some-aws-host"
+        [frontends.web.maxconn]
```

Comment:
It's tested for the [other backend](https://github.com/Nitro/traefik/pull/8/files#diff-86ca9870bc7c46fe2a53d6d4e34419fcR11) defined in this file. See [here](https://github.com/Nitro/traefik/pull/8/files#diff-5abea59f375c71eb653d79df86ed3288R155).

I was considering to create a separate TOML for that specific case, but I think it wouldn't add much value.